docs/api-docs/analytics/bodl-overview.md

```diff
@@ -1,6 +1,8 @@
-# Big Open Data Layer (BODL) - Open Beta
+# Big Open Data Layer (BODL) 
 
-This functionality is currently in beta. Share your feedback with us using the [Partner Portal](https://partners.bigcommerce.com/). 
+<!-- theme: info -->
+> #### Analytics tracking scripts
+> BigCommerce has enabled a data layer to allow developers to create analytics scripts to access data to track shopper behavior. We are developing this feature in the open to obtain your feedback. You can share your feedback with us using the [Partner Portal](https://partners.bigcommerce.com/). 
```

Comment:
Since we haven't enable a built-in data layer yet (that comes when we add this BODL functionality into our platform natively), I wonder if we should open with a description of why this manual creation of BODL is recommended?

So, changing this to something like: 
This guide is meant to help inject the right contextual data into analytics scripts that load on BigCommerce storefronts. It's based what is available today, while incorporating our current thoughts on enabling a data layer as a native BigCommerce feature. We are developing this approach in the open to obtain your feedback! Please share any feedback on BODL with us using the [Partner Portal](https://partners.bigcommerce.com/).